src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,69 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { ChangeEvent, FC, useEffect, useState } from 'react';
+import { ITodoFilter } from '../../types/Todo';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface IProps {
+  setTodosFilter: (filter: ITodoFilter) => void;
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: FC<IProps> = ({ setTodosFilter }) => {
+  const [status, setStatus] = useState('');
+  const [filter, setFilter] = useState('');
```

Comment:
for what?
u already have these states in the `App` component. u can pass it as a prop